src/sr/main.rs

```diff
@@ -143,13 +145,18 @@ where
     let mut role = None;
     let mut task = None;
     let mut user: Option<SUserType> = None;
+    let mut group: Option<SGroups> = None;
     let mut env = None;
+
     while let Some(arg) = iter.next() {
         // matches only first options
         match arg.as_ref() {
             "-u" | "--user" => {
                 user = iter.next().map(|s| escape_parser_string(s).as_str().into());
             }
+            "-g" | "--group" => {
+                group = iter.next().map(|s| escape_parser_string(s).as_str().into());
```

Comment:
need to split comma to get the Vec<<String>>
Like that :
```suggestion
                group = iter.next().map(|s| escape_parser_string(s).as_str().split(",").into());
```